internal/handlers/pg/msg_find.go

```diff
@@ -133,7 +133,11 @@ func (h *Handler) MsgFind(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, er
 
 	var cursorID int64
 
-	if h.EnableCursors {
+	if h.EnableCursors &&
+		!params.SingleBatch &&
+		int64(len(resDocs)) > params.BatchSize {
```

Comment:
```suggestion
	if h.EnableCursors && !params.SingleBatch && int64(len(resDocs)) > params.BatchSize {
```

I don't know why we started to format conditions like that, but let's not do that